content/docs/how-we-work/baseline/logging-details.md

```diff
@@ -0,0 +1,98 @@
+
+### Creating Tickets from Alerts
+
+1. **Review the Alert Details:**
+   - Access the alert details page to view detailed information, including stack traces, affected users, and the frequency of occurrences. This detailed context helps in understanding the issue better and aids in faster resolution.
+   - **Error Context:** Review the contextual data provided by Sentry, such as the environment (production, staging), user actions leading to the error, and system state. This information is critical for diagnosing and resolving issues efficiently. For more details, refer to the [Sentry Issue Details Documentation](https://docs.sentry.io/product/issues/issue-details/).
+
+2. **Integrate Sentry with Your Ticketing System** (e.g., Jira, GitHub Issues):
+   - Select the ticketing system (in our case commonly - Jira) in the **Project Settings** > **Integrations** to integrate Sentry with your ticketing system. Follow the prompts to authenticate and authorize Sentry to access your Jira instance. It is vital to Configure the integration settings, such as default project and issue type, to streamline ticket creation. For more details, refer to the [Sentry Jira Integration Documentation](https://docs.sentry.io/product/integrations/issue-tracking/jira/).
+
+3. **Create a Ticket:**
+   - **Manually Creating a Ticket:**
+     - **Create an Issue** in the alert details view to manually create a ticket by writing all relevant information provided by Sentry to ensure the issue is well-documented.
+
+   - **Automatically Creating Tickets:**
+     - Configure automatic ticket creation in **Project Settings** > **Alerts** by setting up alert rules to trigger ticket creation by specifying the conditions (e.g., error type, frequency) and actions (create a Jira ticket) for automatic ticket creation.
+
+4. **Managing Tickets in Jira:**
+   - Ensure tickets are assigned to the appropriate team or individual in Jira. Use Jira’s workflow management tools to track the progress of the ticket from creation to resolution.
+   - **Tracking and Reporting:** Use Jira’s reporting features to track the status of logged issues. Generate reports to identify trends, recurring issues, and areas for improvement.
+   - **Comments and Updates:** Utilize Jira’s commenting system to add updates, discuss potential solutions, and collaborate with team members directly within the ticket.
+
+## Best Practices
+
+We apply these best practices to ensure effective logging, monitoring, and incident management in our Drupal applications:
+
+- Log significant user actions and interactions using Drupal's built-in functions and custom modules.
+- Enable and review database query logging using `settings.php` and the Database Logging module (`dblog`).
+- Log all API requests, responses, and third-party service interactions using `Guzzle` middleware and capturing successful and failed requests.
```

Comment:
Let's not mention logging *all* requests. It's not worth logging everything and we should not encourage that.